### PR TITLE
Use shellspec docker

### PR DESCRIPTION
**Pull request by willmarks**

### CODE REVIEW

File: test/shellspec-installer.sh [hunk 1]
```diff
@@ -0,0 +1,281 @@
+#!/bin/sh
```
[the remark this answers is not in the document]
Comment (tovbinm) [reply test/shellspec-installer.sh:1]: how about we just use a docker ? https://github.com/shellspec/shellspec/blob/master/docs/docker.md

**Review by ypc-faros — APPROVED** [review]

File: test/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FROM shellspec/shellspec:kcov
+RUN apk add --no-cache curl jq gawk sed
+COPY ./ /src
```
Comment (tovbinm) [inline test/Dockerfile:3]: why do you need to copy the `src/` folder?

Comment (willmarks) [reply test/Dockerfile:3]: I dont! thanks

**Review by tovbinm — APPROVED** [review]

**Comment by tovbinm** [comment]
thanks!

**Pull request merged**